packages/react/src/patterns/sections/LeadSpace/LeadSpace.js

```diff
@@ -90,43 +131,72 @@ const LeadSpace = ({
   image,
   theme,
   title,
+  type,
   variation,
-}) =>
-  featureFlag(
-    DDS_LEADSPACE,
+}) => {
+  const [windowWidth, setWindowWidth] = useState(window.innerWidth);
+  const isMobile = windowWidth <= 671;
```

Comment:
we can also use carbon's breakpoint values here too like in the `Image` component
```
import { baseFontSize, breakpoints } from '@carbon/layout';

....

  const isMobile = windowWidth <= parseFloat(breakpoints.sm.width * baseFontSize);
```

...